standards/SPIFFE_Trust_Domain_and_Bundle.md

```diff
@@ -197,6 +220,38 @@ Trust bundle #2 for example.com:
 
 In trust bundle #2, note that the `spiffe_sequence` parameter has been incremented and the second root certificate for `example.com` has been added. Once this new trust bundle is published and distributed, validators will accept SVIDs signed by either the original or the replacement root certificate. By publishing the replacement certificate well ahead of the expiration of the original certificate, validators have ample opportunity to refresh the trust bundle of example.com and learn of the pending replacement certificate.
 
+## Appendix B. SPIFFE Bundle Map Example
+The following is a sample SPIFFE Bundle Map:
+
+```
+{
+  “trust_domains”: {
+    "example.com": {
+      “sequence_number”: 12035488,
```

Comment:
Should this be the `sequence_number` or the `spiffe_sequence`?

Reply:
It should be `spiffe_sequence`